ui/page/components/wallet_sync_info.go

```diff
@@ -462,6 +462,8 @@ func (wsi *WalletSyncInfo) progressStatusDetails() (int, string) {
 func (wsi *WalletSyncInfo) layoutAutoSyncSection(gtx C) D {
 	return layout.Flex{}.Layout(gtx,
 		layout.Rigid(func(gtx C) D {
+			autoSync := wsi.wallet.ReadBoolConfigValueForKey(sharedW.AutoSyncConfigKey, false)
+			wsi.syncSwitch.SetChecked(autoSync)
```

Comment:
Why is this being done here? I think it should be done in `OnNavigatedTo()`?

Reply:
The value might have been updated in a gorutine and it won't reflect in the UI until another `OnNavigatedTo()` is called.